browser/brave_wallet/eth_allowance_manager_unittest.cc

```diff
@@ -3,10 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/components/brave_wallet/browser/eth_allowance_manager.h"
-
 #include <map>
 #include <memory>
+#include <optional>
 #include <string_view>
 #include <utility>
```

Comment:
Yes, unfortunately because the test name doesn't match the name of the tested file, the existing chromium automation is unable to tell that it should leave that entry separate from the others, unfortunately.